### ISSUE TITLE
Possible issue with environment detection

### ISSUE THREAD

**Issue by SelrahcD** [Evaneos/silex-rest-skeleton#11]
The use of an environment variable named `SILEX_SKT_ENV` for every app using the skeleton might be an issue if all apps are running on the same host but are not sharing the env.
I guess this is not the only env var used.

**Comment by jjsaunier** [comment]
Yes we should make a documentation about that but yes,env variable is proper of each application, it's just a placeholder.

**Comment by ghost** [comment]
Then, we should parameterize the name of the two env variables

**Comment by ghost** [comment]
On the other hand, having a global env var is quite handy too... so that, on a same server, every app has the same env name key, that can be useful... but dangerous... ;)
